apps/web/src/services/cost-explorer/CostExplorerLNB.vue

```diff
@@ -58,15 +65,8 @@ const state = reactive({
             label: i18n.t(MENU_INFO_MAP[MENU_ID.COST_EXPLORER_COST_ANALYSIS].translationId),
         },
         {
-            type: MENU_ITEM_TYPE.DROPDOWN,
-            id: DATA_SOURCE_DROPDOWN_KEY,
-            selectOptions: {
-                items: costQuerySetState.dataSourceList.map((dataSource) => ({
-                    name: dataSource,
-                    label: dataSource,
-                })),
-                defaultSelected: state.selectedDataSource,
-            },
+            type: MENU_ITEM_TYPE.SLOT,
+            id: 'data-source-dropdown',
```

Comment:
nitpick: Since the `id` is hardcoded, it might be better to turn it into a variable.

Reply:
Thanks!